### DISCUSSION TITLE
App Icon 2.0: Selecting the right shape (not colors!)

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Notes#261] [Polls]
@TriliumNext/everyone ,

Thank you for your votes in #203 .

Based on the different discussions now we can further refine the icon.

**Important: Don't mind the color scheme, that can be changed and we can vote it once we have the shape fixed in.*

1. We have the former almost-winner at 50%:

![image](https://github.com/user-attachments/assets/fb1f2d92-5c36-4131-b0b4-0cf9e09c44cc)

---

2. We have the first design from @jerakin (with purple variant as well):

![image](https://github.com/user-attachments/assets/48e83e5a-ffeb-420f-b853-27474b66ed1a)

---

3. And a new contender from my designer friend:

![image](https://github.com/user-attachments/assets/3ec9cf52-373a-4af7-a201-bf03158afce0)

---

4. And the second design from @jerakin (see [additional versions](https://github.com/orgs/TriliumNext/discussions/203#discussioncomment-10104308)):

![image](https://github.com/user-attachments/assets/f38617e5-f6cc-4a5f-a93d-e70a62298f05)

---

5. And last but not least, @meichthys 's simplified design:

![image](https://github.com/user-attachments/assets/e7f17d40-f387-4809-bec3-51e1f6558839)

**Comment by zerebos** [comment]
These are all very good. I'd love to see the 3rd one of option 3 but using the original colors.

**Reply by eliandoran** [reply]
@rauenzi , that can certainly be arranged.

**Reply by eliandoran** [reply]
If we get that icon, I'll certainly use the purple one for dev/nightly builds. 👿

**Reply by alexpietsch** [reply]
I agree, I would also prefer a design that is different from Trilium's original logo.

**Reply by eliandoran** [reply]
@rauenzi ,

![image](https://github.com/user-attachments/assets/0b65dcbf-c89b-4834-ad57-494427aa5240)

**Reply by zerebos** [reply]
I absolutely love the one on the right honestly

**Reply by Nriver** [reply]
Tri-lemon Notes : D

**Reply by LarsaSara** [reply]
Ohh... My son(design) started to cry, it's a flower, not a lemon.

**Reply by haganjoe** [reply]
As a big fan of purple and black, I prefer option 3rd option of 3rd sample with deep purples. Sorry, not a fan of the original colors lol :(

**Reply by zerebos** [reply]
I would just worry about the usage of purple overlapping with Obsidian's identity.

**Reply by eliandoran** [reply]
@rauenzi , Obsidian doesn't have (or shouldn't have) a monopoly on the color purple. :)

**Reply by zerebos** [reply]
Of course not, but going all-in on purple as a brand and accent color for a note-taking app specifically can lead to the public perception that we are trying to copy or rip them off. Not anything in terms of legality of course.

**Reply by TechToch** [reply]
I would say that the purple is trendy, nice color

**Reply by TechToch** [reply]
Otherwise stay identical ? 1

**Reply by meichthys** [reply]
lol, I thought this whole poll was just to pick the shape, not the color.

**Comment by meichthys** [comment]
## Could i try one more? (Colors and shading could be changed/removed)
![image](https://github.com/user-attachments/assets/de16560f-9689-41d0-995a-e9da7eb04a45)

## And for all of our Obsidian fanboys (Joking!) out there 😝:
![image](https://github.com/user-attachments/assets/f678ec7c-b426-41de-9560-99431f9aeda9)

**Reply by zerebos** [reply]
I like the third one perhaps with less of a gradient into black.

**Reply by meichthys** [reply]
I also thought the third one was the best of the three.

**Reply by TechToch** [reply]
can you make it cyberpunk purple ? ;-)

**Reply by diyoyo** [reply]
I'd vote for the 3rd one too. Honestly, all the other options mentioned by the @eliandoran will raise some criticisim regarding misplaced connotations. I'm sorry I missed the topic #203 (and so many others, I've been a bit overwhelmed lately).

Some poetic interpretation about logo number 3 (which may already have been zadam's reasons?):
- the curves remind me of a rotation of the ex-leaves, so we're in a cycle
- the colors (and the fact that these were leaves in the past) are the colors of leaf life stages.
- combining these two points means that it's all about the cycle of life, any note we take is connected to the rest, and if we don't know yet what it's purpose is, it will be recycled later to generate new knowledge. So the leaf starts green, dry out from red (Fall) to brown, but never dies.

**Reply by Alumniminium** [reply]
i'd be very happy with the 3rd

**Reply by meichthys** [reply]
> * the curves remind me of a rotation of the ex-leaves, so we're in a cycle

That's what I was shooting for 👍

Also the darkened middle reminds me of ideas that have not yet been expressed (or noted), while the brighter leaf tips represent ideas or leaf notes in trilium which are continuously being more completely expressed, but that still relate back to the other original ideas that were once just a figment of your imagination.

**Reply by meichthys** [reply]
For all of our Obsidian fanboys (Joking!) out there 😝:
![image](https://github.com/user-attachments/assets/f678ec7c-b426-41de-9560-99431f9aeda9)

(Also updated in initial comment)

**Reply by diyoyo** [reply]
> For all of our Obsidian fanboys (Joking!) out there 😝: ![image](https://private-user-images.githubusercontent.com/10717998/351745706-f678ec7c-b426-41de-9560-99431f9aeda9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MzI1MTYsIm5iZiI6MTcyMTgzMjIxNiwicGF0aCI6Ii8xMDcxNzk5OC8zNTE3NDU3MDYtZjY3OGVjN2MtYjQyNi00MWRlLTk1NjAtOTk0MzFmOWFlZGE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI0VDE0NDMzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ4NmE5MWM4Yjc5ZjRiZTlmODE4ZDA1ZDA4ZTg2MmRjNDE5NzI0OTQ2NTA3ZjVlNWY5YWQzZTkzODViMDY0ODImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UPd_Xi3KUEsF6Va52OfZs8CSrE_QAnbROwKs9-q1QeU)
> 
> (Also updated in initial comment)

Could anyone teach me the main differences between Obsidian and Trilium? I actually never tried Obsidian, but the name rings a bell, so please educate me.

**Reply by zerebos** [reply]
Current Obsidian as compared to current Trilium from a user perspective (not a dev perspective):
- Hugely popular
- Large dev community (1,000+ plugins)
- Uses markdown as primary note-taking method
- Fancy nearly WYSIWYG using markdown via a custom editor (see image below, all components there you can click into to reveal markdown)
- Custom editor has support for a lot of advanced options including advanced table support
- Very extensible via plugins and themes, plugins can add even more custom component types to the editor
- Blocks in editor are collapsible
- Uses files on the filesystem (no database) so the tree you see in-app matches your filesystem
- More cohesive and powerful UI
- Much more customization and settings out of the box
- "Autolinks" when the app thinks you meant to link to another note due to text having the exact title of the note
- Audio recorder built in
- Has translations
- Has mobile apps
- No web version
- Closed source
- No free sync by default (there are community plugins that do it)
- No free web share
- No equivalent of trilium clone notes
- No real equivalent of trilium render notes (you can do it on a block-level see graph in image below)
- Less powerful (but more user-friendly) attribute and searching system
- No real equivalent of promoted attributes (some community plugins help)
- Only markdown and canvas note types
- Folders are only folders, not notes (there are community plugins to do this)



![image](https://github.com/user-attachments/assets/36759987-f78b-4b36-9bfa-56aca8ca14a4)

**Reply by diyoyo** [reply]
> Current Obsidian as compared to current Trilium from a user perspective (not a dev perspective):
> 
> * Hugely popular
> * Large dev community (1,000+ plugins)
> * Uses markdown as primary note-taking method
> * Fancy nearly WYSIWYG using markdown via a custom editor (see image below, all components there you can click into to reveal markdown)
> * Custom editor has support for a lot of advanced options including advanced table support
> * Very extensible via plugins and themes, plugins can add even more custom component types to the editor
> * Blocks in editor are collapsible
> * Uses files on the filesystem (no database) so the tree you see in-app matches your filesystem
> * More cohesive and powerful UI
> * Much more customization and settings out of the box
> * "Autolinks" when the app thinks you meant to link to another note due to text having the exact title of the note
> * Audio recorder built in
> * Has translations
> * Has mobile apps
> * No web version
> * Closed source
> * No free sync by default (there are community plugins that do it)
> * No free web share
> * No equivalent of trilium clone notes
> * No real equivalent of trilium render notes (you can do it on a block-level see graph in image below)
> * Less powerful (but more user-friendly) attribute and searching system
> * No real equivalent of promoted attributes (some community plugins help)
> * Only markdown and canvas note types
> * Folders are only folders, not notes (there are community plugins to do this)
> 
> ![image](https://private-user-images.githubusercontent.com/6865942/351764051-36759987-f78b-4b36-9bfa-56aca8ca14a4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MzYwMDIsIm5iZiI6MTcyMTgzNTcwMiwicGF0aCI6Ii82ODY1OTQyLzM1MTc2NDA1MS0zNjc1OTk4Ny1mNzhiLTRiMzYtOWJmYS01NmFjYThjYTE0YTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjRUMTU0MTQyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzVkZGE3NTdhM2M2ZDhmMDI4YzkxMjRiMmZiNjAwMzJlZDAwYzQxZjA5ZGEyY2E5ZGI2MzRmNDYxODY1Mzc2MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.LkCUAXkRIFAeSdZ3CoJK5h-9RuVJEgjLKWiD9M1yL78)

Ok, you're very good at selling the stuff 😅 At first, I started to think "why the hell am I spending so much time on Trilium", and then, the more I read, the more I understood 🤣. Actually, you had me at "closed source". I looked at the Obsidian website and got tricked when reading "Always free". I thought it was like the others : an open-source project that has a premium pricing plan, but forgot to check whether it was open source .

THanks for taking the time 👍

**Reply by meichthys** [reply]
> ## Could i try one more? (Colors and shading could be changed/removed)
> 
> ![image](https://private-user-images.githubusercontent.com/10717998/351342807-de16560f-9689-41d0-995a-e9da7eb04a45.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MzgxMDAsIm5iZiI6MTcyMTgzNzgwMCwicGF0aCI6Ii8xMDcxNzk5OC8zNTEzNDI4MDctZGUxNjU2MGYtOTY4OS00MWQwLTk5NWEtZTlkYTdlYjA0YTQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI0VDE2MTY0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE2YWQzMjdlMDcwNzFjZDE5YTk0NWJkODcwZmQ3MTFkMmVjZjAyMjBlYjNmNzMzMDI2OTk4MjAwZjNjY2IxMDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PJu-eVwGb3wJM-MnGMBb04uYWkA6MIqD-cIEqb7qlYQ)
> ## And for all of our Obsidian fanboys (Joking!) out there 😝:
> 
> ![image](https://private-user-images.githubusercontent.com/10717998/351745706-f678ec7c-b426-41de-9560-99431f9aeda9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MzgxMDAsIm5iZiI6MTcyMTgzNzgwMCwicGF0aCI6Ii8xMDcxNzk5OC8zNTE3NDU3MDYtZjY3OGVjN2MtYjQyNi00MWRlLTk1NjAtOTk0MzFmOWFlZGE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI0VDE2MTY0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxNGFkNDQ5ZmY2ZmVjODUwMGM1ZDg4YjdlOTYzMTI4N2YzOTFiYWI0ODg0ZjZhNDA1MzY1ZWVjMDNmNDMzMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3dvgy1UhmWjW2yE5ak9I4s6K6yPQTfGGF09ypUXSSCY)

@eliandoran could we swap these newer logs in for # 5 or add a new option to the poll?

**Reply by CrO2Cl2** [reply]
@meichthys I dont know if it is just me, but the images you send are not loading

**Reply by eliandoran** [reply]
@meichthys , @CrO2Cl2 is right, the images in your last comment are not accessible.

**Reply by meichthys** [reply]
Interesting.. they were just the same ones here:
https://github.com/orgs/TriliumNext/discussions/261#discussioncomment-10126634

**Reply by zerebos** [reply]
It's a github issue when quoting posts with images currently, just scroll up to the originally quoted post and the images should be fine.

**Reply by eliandoran** [reply]
@meichthys , I will remake the poll however this will be the final poll as we have been moving the goalposts a few times. I'll try to make the poll also mention colors.

**Reply by meichthys** [reply]
@eliandoran I don't think you need to re-make it - just edit the poll to show this design instead of my other one perhaps?

**Reply by eliandoran** [reply]
@meichthys , that could work but unless we reset the vote then poll would no longer be technically valid, since people voted for the old one.

**Reply by eliandoran** [reply]
Anyway we'll soon need to refine the icon selection based on color so it's OK to have a new, final poll

**Reply by meichthys** [reply]
People can change their votes if they liked my design.

**Comment by eliandoran** [comment]
@TriliumNext/everyone , thank you for your votes.

Final poll has dropped in, please take your time to vote: #272 .
